apps-devstg/us-east-1/base-network/config.tf

```
@@ -10,7 +10,7 @@ provider "aws" {
 # Backend Config (partial)    #
 #=============================#
 terraform {
-  required_version = "~> 1.2.7"
+  required_version = "~> 1.3.5"
```

Lately we favor the following:
```suggestion
  required_version = "~> 1.3"
```

---

@crcedenop I saw your comment with the terraform output and it looks like the suggested version constraint doesn't break the plan. Would that be enough to accept and commit my suggestion above? If you agree, then I can resolve this conversation and approve the PR.